felix/bpf-gpl/skb.h

```diff
@@ -68,6 +68,10 @@ static CALI_BPF_INLINE void skb_refresh_start_end(struct cali_tc_ctx *ctx) {
 static CALI_BPF_INLINE long skb_iphdr_offset(void)
 {
 	if (CALI_F_IPIP_ENCAPPED) {
+		if (GLOBAL_FLAGS & CALI_GLOBALS_IPIP_NO_OUTER_HEADERS) {
```

Comment:
I wonder if instead we would be able to just load the L3 (`CALI_F_L3`) programs rather than handling it as a runtime configuration. Then the change could be limited to felix only :thinking:

Reply:
Right. let me check.